### PR TITLE
patch-1

### PR DESCRIPTION
**Pull request by leonbreukelman**
Add PUID and PGID
Update permissions 
Touch a file
Update description

### CODE REVIEW

**Comment by zoemdoef** [comment]
Add PUID and PGID
Update permissions
Touch a file
Update description

File: run.sh [hunk 1]
```diff
@@ -104,6 +105,7 @@ if [[ ! -f /var/spool/cron/crontabs/pi ]]; then
   mv /root/crontab /var/spool/cron/crontabs/pi
 fi
 touch /etc/crontab /etc/cron.d/* /var/spool/cron/crontabs/*
+touch /etc/cron.d/emoncms
```
Comment (jakezp) [inline run.sh:108]: Duplication of line 107

Comment (leonbreukelman) [reply run.sh:108]: Line 107 does not create the emoncms file

File: README.md [hunk 2]
```diff
@@ -25,3 +26,13 @@ Change:
               MYSQL_PASSWORD - MySQL password
               MQTT_HOST - MQTT hostname or IP (If MQTT_HOST is not specified, MQTT support will not be enabled)
               /tmp - preferred location on the host
+              
+              User / Group Identifiers
```
Comment (jakezp) [inline README.md:30]: Not sure if it's required for this image...
Update, removing preceding whitespace...

File: Dockerfile [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Dockerfile for base image for emoncms
 FROM ubuntu:14.04
 
-MAINTAINER jakezp
+MAINTAINER zoemdoef
```
Comment (jakezp) [inline Dockerfile:4]: ???

**Review by jakezp — CHANGES_REQUESTED** [review]
Comments added.

File: run.sh (hunk 1 above)
Comment (jakezp) [reply run.sh:108]: /etc/cron.d/emoncms not required. User 'pi' cronjobs listed in /var/spool/cron/crontabs/pi - created in line 105 of it does not exist yet.

File: Dockerfile (hunk 3 above)
Comment (leonbreukelman) [reply Dockerfile:4]: Not sure how that ended up here

**Pull request closed without merging**